src/io/files.js

```diff
@@ -1249,35 +1249,71 @@ p5.PrintWriter = function(filename, extension) {
    * @method write
    * @param {Array} data all data to be written by the PrintWriter
    * @example
-   * <div class="norender notest">
+   * <div>
    * <code>
-   * // creates a file called 'newFile.txt'
-   * let writer = createWriter('newFile.txt');
-   * // write 'Hello world!'' to the file
-   * writer.write(['Hello world!']);
-   * // close the PrintWriter and save the file
-   * writer.close();
+   * function setup() {
+   *    createCanvas(100, 100);
+   *    background(234, 40, 95);
```

Comment:
for simplicity, you can remove these first two lines and a 100x100 canvas will be autocompleted